### PR TITLE
flow: introduces bulk flow sending using flow message

### PR DESCRIPTION
**Pull request by safchain**

### CODE REVIEW

File: flow/server/flow_server.go [hunk 1]
```diff
@@ -99,25 +99,29 @@ func (c *FlowServerWebSocketConn) OnMessage(client ws.Speaker, m ws.Message) {
...
-	if len(c.ch) >= c.maxFlowBufferSize {
-		c.numOfLostFlows++
-		if c.timeOfLastLostFlowsLog.IsZero() ||
-			(time.Now().Sub(c.timeOfLastLostFlowsLog) >= time.Second) {
-			logging.GetLogger().Errorf("Buffer overflow - too many flow updates, removing and not storing flows: %d", c.numOfLostFlows)
-			c.timeOfLastLostFlowsLog = time.Now()
-			c.numOfLostFlows = 0
+	logging.GetLogger().Debugf("New flow message from Websocket connection: %+v", msg)
+
+	// TODO(safchain) handle mutliple type of message
```
Comment (safchain) [inline flow/server/flow_server.go]: @masco-enovance here where we could handle type of message

**Comment by safchain** [comment]
run tests

**Comment by masco-enovance** [comment]
run skydive-functional-tests-backend-elasticsearch

**Comment by masco-enovance** [comment]
run skydive-functional-tests-backend-elasticsearch

**Comment by safchain** [comment]
run skydive-cdd-overview-tests

**Pull request merged**